vmc/data_source_vmc_sddc.go

```diff
@@ -212,9 +212,14 @@ func dataSourceVmcSddcRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("nsxt_ui", *sddc.ResourceConfig.Nsxt)
 		if sddc.ResourceConfig.NsxCloudAdmin != nil {
 			d.Set("nsxt_cloudadmin", *sddc.ResourceConfig.NsxCloudAdmin)
-			d.Set("nsxt_cloudadmin_password", *sddc.ResourceConfig.NsxCloudAdminPassword)
+			// Evade nil pointer dereference when user's access_token doesn't have NSX roles
+			if sddc.ResourceConfig.NsxCloudAdminPassword != nil {
+				_ = d.Set("nsxt_cloudadmin_password", *sddc.ResourceConfig.NsxCloudAdminPassword)
```

Comment:
nit: do you need the `_ =`? I see line 212 does not have it.

Reply:
The IDE complains I'm not using the result ot d.Set and presents it as a warning. Line 212 doesn't have it as I want to limit the changes on the side. Possibly there will be a dedicated change to solve all the warnings.